src/masternode-sync.cpp

```diff
@@ -324,12 +324,12 @@ void CMasternodeSync::ProcessTick()
         if(Params().NetworkIDString() == CBaseChainParams::REGTEST)
         {
             if(nRequestedMasternodeAttempt <= 2) {
-                pnode->PushMessage(NetMsgType::GETSPORKS); //get current network sporks
+                g_connman->PushMessageWithVersion(pnode, INIT_PROTO_VERSION, NetMsgType::GETSPORKS); //get current network sporks
```

Comment:
Do we know that this always runs before the peer handshake has been done ?

I would think that the order is actually indeterminate here and it seems re-sending the version could get us disconnected.